Runtime/Components/WorldGen.cs

```diff
@@ -198,14 +202,14 @@ private void BuildMesh(int Tx, int Ty, GameObject HoldsMesh)
 		}
 	}
 
-
-    private float getRatio(float pX,float pY,out float height)
+    private float getRatio(float pX, float pY, out float height)
     {
 		height = 0;
-		if (UseCityGrid)
+		if (SpawnPOIs && GetComponent<POI_Gen>() != null)
 		{
+			TerrainNoiseModifier[] Mods =GetComponent<POI_Gen>()?.Mods;
 			float ratio = 0;
-			foreach(TerrainNoiseModifier TNM in Modifiers)
+			foreach(TerrainNoiseModifier TNM in Mods)
```

Comment:
The closeness of Mods and Modifies has already caused issues, are both still present in WorldGen?